_docs/pql_queries.md

```diff
@@ -36,13 +36,13 @@ You can submit your own Queries by editing [voxpupuli.github.io/_docs/pql_querie
 
 ### List all certificate names from all known nodes
 
-```
+```console
 puppet query 'nodes[certname] {}'
 ```
 
 Result:
 
-```
+```json
 [
   {
     "certname": "puppetserver.bastelfreak.org"
```

Comment:
Unrelated, but probably best to use `example.org` instead.

Reply:
Changed to `example.org`.